rj_gameplay/classifier/fumble_classifier.py

```diff
@@ -0,0 +1,96 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+
+
+def fumble_classifier(world_state: rc.WorldState, possess) -> None:
```

Comment:
You have this in a few places, so I'll mention it at the top, you should be putting a type next to most and putting the correct return type for each function. For example in this case the function declaration should be:
`def fumble_classifier(world_state: rc.WorldState, possess: int) -> bool`